local-storage-manager/src/main/java/com/stnetix/ariaddna/localstoragemanager/LocalService/LocalService.java

```diff
@@ -0,0 +1,10 @@
+package com.stnetix.ariaddna.localstoragemanager.LocalService;
```

Comment:
the package name need set in small char com.stnetix.ariaddna.localstoragemanager.localservice;